gh-dependabot

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3
+
+import click
+import shutil
+import subprocess
+import json
+import csv
+
+@click.group()
+def dependabot():
+    """A GH CLI extension to manage dependabot"""
+
+@dependabot.command()
+@click.argument('repo', nargs=-1)
+@click.option('-o', '--output', required=True, help='Path to the output file')
+def export(repo, output):
+    """Pulls all dependabot alerts and exports them to a CSV file
+    
+       REPO is space separated in the OWNER/NAME format
+    """
+    alerts = []
+    for repository in repo:
+        click.echo("Exporting dependabot alerts from %s into %s" % (repository, output))
+
+        alerts += get_dependabot_alerts(repository)
+    
+    generate_csv(alerts, output)
+    
+
+def get_dependabot_alerts(repo_name, cursor='null'):
+    query = 'query ($org: String! $repo: String! $cursor: String){ \
+      repository(owner: $org name: $repo) { \
+        name \
+        vulnerabilityAlerts(first: 100 after: $cursor) { \
+          pageInfo { \
+            hasNextPage \
+            endCursor \
```

Comment:
Are the backslashes necessary?

Reply:
So my only option was to make this one gigantic line but for visibility reasons, I decided to break it out into multiple lines. The back slashes were necessary to ensure that all the lines were associated together. Otherwise python would think that they aren't part of the query 😞

Reply:
You can use multiline strings instead of backslashes and it should respect the indentation

```python
a = """
{
    "a":"b"
}
"""
```

Reply:
Thanks for that! I updated my query to use the multiline string instead and now it looks a lot better 😄